bitnami/grafana/templates/deployment.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "grafana.fullname" . }}
+  labels: {{ include "grafana.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels: {{ include "grafana.matchLabels" . | nindent 6 }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "grafana.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        helm.sh/chart: {{ include "grafana.chart" . }}
+{{- if .Values.podAnnotations }}
+      annotations:
+         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+         checksum/dashboard-provider: {{ include (print $.Template.BasePath "/dashboard-provider.yaml") . | sha256sum }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+{{- end }}
+    spec:
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+      {{- end }}
+{{- include "grafana.imagePullSecrets" . | indent 6 }}
+     {{- if .Values.securityContext.enabled }}
+      securityContext:
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
+      {{- end }}
+{{- with .Values.nodeSelector }}
+      nodeSelector:
+{{-  tpl (toYaml .) $ | indent 8 }}
+{{- end }}
+{{- with .Values.affinity }}
+      affinity:
+{{-  tpl (toYaml .) $ | indent 8 }}
+{{- end }}
+{{- with .Values.tolerations }}
+      tolerations:
+{{- tpl (toYaml .) $ | indent 8 }}
+{{- end }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        envFrom:
+        - configMapRef:
+            name: "{{ include "grafana.fullname" . }}-envvars"
+        - secretRef:
+            name: "{{ include "grafana.fullname" . }}-secret"
+        {{- if .Values.smtp.existingSecret }}
+        - name: GF_SMTP_USER
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.smtp.existingSecret }}
+              key: {{ .Values.smtp.existingSecretUserKey | default "user" }}
+        - name: GF_SMTP_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.smtp.existingSecret }}
+              key: {{ .Values.smtp.existingSecretPasswordKey | default "password" }}
+        {{- end }}
+{{- if .Values.extraEnvVars }}
+{{ toYaml .Values.extraEnvVars | indent 8 }}
+{{- end }}
```

Comment:
It seems we have an environment variable to install plugins on 1st run. Shouldn't we add support for it here? https://github.com/bitnami/bitnami-docker-grafana#install-plugins-at-initialization

Reply:
It is on the ConfigMap from all the env vars are loaded. Also there is a parameter on the values called `plugins` to set the names separated by comas.